jobs/cloud_controller_ng/templates/cloud_controller_ng.yml.erb

```diff
@@ -451,6 +451,15 @@ rate_limiter:
 
 max_concurrent_service_broker_requests: <%= p("cc.max_concurrent_service_broker_requests") %>
 
+<% instances = p("cc.rate_limiter_v2_api.enabled") ? link("cloud_controller").instances.length : 1 %>
```

Comment:
We now have this temporary `instances` variable twice (here and above for the 'normal' rate limiter. What about putting this into the top `<% ... %>` block and changing it to `cc_instances = link("cloud_controller").instances.length`?

Reply:
Next time 🙈